examples/models/efficient_sam/README.md

```diff
@@ -0,0 +1,35 @@
+# EfficientSAM Model Export
+
+This example demonstrates how to export the [EfficientSAM](https://github.com/yformer/EfficientSAM) model to Core ML and XNNPACK using ExecuTorch.
+
+# Instructions
+
+## 1. Setup
+
+Follow the [tutorial](https://pytorch.org/executorch/main/getting-started-setup#) to set up ExecuTorch.
+
+## 2. Exports
```

Comment:
Are there any performance numbers of those two backend? Would be great to list them here.

Reply:
I have just listed some performance numbers.